src/com/georgidinov/domain/record/Record.java

```diff
@@ -0,0 +1,96 @@
+package com.georgidinov.domain.record;
+
+import com.georgidinov.domain.book.Book;
+import com.georgidinov.domain.person.Person;
+
+import java.time.LocalDate;
+
+import static com.georgidinov.util.MyDateFormatter.getDateString;
+import static com.georgidinov.util.MyMessages.*;
+
+public class Record {
+
+    public static final int REGULAR_BORROW_PERIOD_IN_DAYS = 14;
+    public static final int MAX_BORROW_PERIOD_IN_DAYS = 28;
+
+    private static int instanceCounter = 0;
+    private int id;
+
+    private int bookId;
+    private int readerId;
+    private LocalDate issuedDate;
+    private LocalDate dueDate;
+    private LocalDate maxPostponeDate;
+
+    public Record(Book book, Person person) {
+        this.id = ++instanceCounter;
+        this.bookId = book.getId();
+        this.readerId = person.getId();
+        this.issuedDate = LocalDate.now();
+        this.dueDate = issuedDate.plusDays(REGULAR_BORROW_PERIOD_IN_DAYS);
+        this.maxPostponeDate = issuedDate.plusDays(MAX_BORROW_PERIOD_IN_DAYS);
+    }
+
+    public int getId() {
+        return id;
+    }
+
+    public int getBookId() {
+        return bookId;
+    }
+
+    public int getReaderId() {
+        return readerId;
+    }
+
+    public LocalDate getIssuedDate() {
+        return this.issuedDate;
+    }
+
+    public String getIssuedDateString() {
+        return getDateString(this.issuedDate);
+    }
+
+    public LocalDate getDueDate() {
+        return this.dueDate;
+    }
+
+    public String getDueDateString() {
+        return getDateString(this.dueDate);
+    }
+
+    public void postponeRequest(int days) {
+        if (this.postponeIsPossible(days)) {
+            this.dueDate = this.dueDate.plusDays(days);
+            System.out.println(POSTPONE_REQUEST + SPACE + GRANTED + COMA +
```

Comment:
make this a literal

Reply:
It's not apparent exactly what you're outputting because of these constants in separate classes.